src/components/Selector/Selector.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react"
+import { Meta, Story } from "@storybook/react/types-6-0"
+
+import Selector, { SelectedItems, SelectorProps } from "./Selector"
+import { mockedOptions } from "./__mocks__/options"
+
+export default {
+  title: "Design System/Selector",
+  component: Selector,
+} as Meta
+
+const Template: Story<SelectorProps> = (props) => <Selector {...props} />
+
+/** Multi selector controlled to make sure the selected items become persistent through state changes */
+const MultiTemplate: Story<SelectorProps> = (props) => {
+  const [selectedItems, setSelectedItems] = useState<SelectedItems>([])
+
+  return (
+    <Selector
+      {...props}
+      selectedItems={selectedItems}
+      onSelectedItemsChange={(newSelectedItems) => {
+        setSelectedItems(newSelectedItems)
+      }}
+    />
+  )
+}
+
+export const Default = Template.bind({})
+
+Default.args = {
+  options: mockedOptions,
+}
+
+export const WithLabel = Template.bind({})
+
+WithLabel.args = {
+  options: mockedOptions,
+  label: "Select an element: ",
+}
+
+export const MultipleSelection = MultiTemplate.bind({})
+
+MultipleSelection.args = {
+  options: mockedOptions,
+  label: "Select one or multiple elements: ",
+  multi: true,
+}
+
+export const Disabled = Template.bind({})
+
+Disabled.args = {
+  options: mockedOptions,
+  label: "Select an element: ",
+  disabled: true,
+}
```

Comment:
Not exactly related to the story code per se, but testing the story I noticed the select input changes color on hover when it's disabled, I imagine there should be no change when it's disabled
![image](https://user-images.githubusercontent.com/12552164/110952432-fd7fbb00-8324-11eb-977f-c63361ed1128.png)